src/Command/UpdateCommand.php

```diff
@@ -4,9 +4,89 @@
 
 namespace App\Command;
 
-class UpdateCommand
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(
+    name: 'app:update',
+    description: 'Check if important commands have been run. Use --force to run the commands.',
+)]
+class UpdateCommand extends Command
 {
-    // TODO: Test that migrations have been run.
-    // TODO: Run test of status for templates. No templates = clean install. Install all?
-    // TODO: Update existing templates.
+    protected function configure(): void
+    {
+        $this->addOption('force', 'f', InputOption::VALUE_NONE, 'Update all requirements. Otherwise, update command only checks requirements.');
```

Comment:
I don't like the naming combination here. `--force` means something else for me. If we need a status command I would make a dedicated `app:status`.

For `aap:update` we could follow doctrine migrations:
* no option: Prompt for "are yoy sure?"
* `--no-interaction`: just run